model/src/main/java/org/cbioportal/model/Gene.java

```diff
@@ -49,4 +51,26 @@ public Integer getLength() {
     public void setLength(Integer length) {
         this.length = length;
     }
+    
+    public String getChromosome() {
```

Comment:
I'm guessing that if we don't want to distinguish this, it is better to mark these as on X rather than Y ... since annotated it as Y might indicate that females would not have the gene, whereas marking it on X correctly indicates that all humans have the gene.